packages/x-components/src/__stubs__/search-response-stubs.factory.ts

```diff
@@ -19,8 +19,13 @@ export function getSearchResponseStub(): SearchResponse {
     partialResults: [],
     promoteds: getPromotedsStub(),
     queryTagging: {
-      params: {},
-      url: ''
+      params: {
```

Comment:
Why do we need this info right now? As far as I understand we're not testing it right?